hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -447,6 +447,8 @@ public void testBlockDeletion() throws Exception {
       // An interval will delete 1 * 2 blocks
       deleteAndWait(svc, 1);
 
+      GenericTestUtils.waitFor(() ->
+              containerData.get(0).getBytesUsed() < 3, 100, 3000);
```

Comment:
Unrelated to your changes. Can we set the block deletion limit to 2 at line 401?
`dnConf.setBlockDeletionLimit(2);`
Then here we should check `containerData.get(0).getBytesUsed() == 100`

Reply:
Thanks @lokeshj1703 - `dfConf.setBlockDeletionLimit` corrected from 3 to 2 for test.  Setting conditional check to equiv 100 (`containerData.get(0).getBytesUsed() == containerSpace / 3`).